src/components/Login.js

```diff
@@ -142,7 +142,25 @@ const Login = (props) => {
           if (res.status === 200) {
             const token = res.data.token;
             const user = res.data.user;
-            appContext.login(user, token, isRemember);
+            // GET logo URL from API if user belongs to an organization
+            // and apply it to logoPath state before completing login
+            if (user.policy.organization) {
+              const STAKEHOLDER_API =
```

Comment:
Can you move this `getStakeholder(id)` query to `api/stakeholders.js` to avoid duplication?

Reply:
I created a `getStakeholder(id)` query in `api/stakeholders.js` which is now being called whenever we need to retrieve the logo URL.
Also, instead of using the IconLogo component on `Login.js` I applied the `logo.svg` directly since we only expect to see the Greenstand logo on that page and it was causing one of the tests to fail.